src/TreeHelper/BeingPassedToHelper.cpp

```diff
@@ -0,0 +1,28 @@
+//
+// Created by jessevw on 19.11.19.
+//
+#include <include/roboteam_ai/bt/composites/Selector.hpp>
+#include <include/roboteam_ai/conditions/IsInDefenseArea.hpp>
+#include <include/roboteam_ai/skills/Receive.h>
+#include <include/roboteam_ai/bt/composites/Sequence.hpp>
+#include "bt/RoleDivider.h"
+#include "bt/tactics/DefaultTactic.h"
+#include "bt/Role.h"
+#include "skills/gotopos/GoToPos.h"
+#include "skills/Attack.h"
+#include "include/roboteam_ai/TreeHelper/BeingPassedToHelper.h"
+
+namespace bt {
+    BeingPassedToHelper::BeingPassedToHelper() {}
+
+    std::shared_ptr<bt::Node> BeingPassedToHelper::createBeingPassedToChecker() {
+        auto localbb = std::make_shared<bt::Blackboard>();
+
+        std::shared_ptr<bt::Sequence> seque = std::make_shared<bt::Sequence>();
+        std::shared_ptr<rtt::ai::Receive> receive = std::make_shared<rtt::ai::Receive>("receivernode", localbb);
+        std::shared_ptr<rtt::ai::IsInDefenseArea> defenseArea = std::make_shared<rtt::ai::IsInDefenseArea>("defendy", localbb);
```

Comment:
I don't like the string naming, but not a huge problem.

Reply:
As in, use names you can recognize 3 months down the line (especially 'defendy' is very vague)